### ISSUE TITLE
1.72b BUG skill summon UI

### ISSUE THREAD

**Issue by DimasjkTV** [suriyun-production/mmorpg-kit-docs#1324]
Fresh install. Singleplayer.
If we use the summoning skill and our character dies, then the icon of the summoned creature with a timer does not disappear. Also, if you log out, then the summoned creature is automatically resurrected.
Also, after the logout, if we re-summon an automatically resurrected creature, then its HP in the UI bar will disappear.
Look at the video.
https://youtu.be/VZkezsyFETs

**Comment by insthync** [comment]
Thank you for bug report, you can fix it by add codes in `BaseCharacterEntity_DamageFunctions`

find for function: `public virtual void Killed(EntityInfo lastAttacker)`
Add
```
            for (int i = summons.Count - 1; i >= 0; --i)
            {
                summons[i].UnSummon(this);
                summons.RemoveAt(i);
            }
```

![Screenshot_827](https://user-images.githubusercontent.com/2697550/144328941-6354cc72-b110-4744-9809-eda22f2bd746.png)

**Comment by DimasjkTV** [comment]
> for (int i = summons.Count - 1; i >= 0; --i)
>             {
>                 summons[i].UnSummon(this);
>                 summons.RemoveAt(i);
>             }

It works. Thanks!

**Comment by insthync** [comment]
Fixed in 1.72c
